support/metrics/metricsTracker.go

```diff
@@ -0,0 +1,179 @@
+package metrics
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"runtime/debug"
+	"time"
+
+	"github.com/stellar/kelp/support/networking"
+
+	"github.com/google/uuid"
+)
+
+var (
+	amplitudeAPIURL string = "https://api2.amplitude.com/2/httpapi"
+)
+
+// Custom events in Amplitude should be named with "ce:event_name",
+// so the web console displays it as "[Custom] event_name".
+const (
+	startupEventName string = "ce:test_startup"
+	updateEventName  string = "ce:test_update"
+	deleteEventName  string = "ce:test_delete"
+)
+
+// MetricsTracker wraps the properties for Amplitude events,
+// and can be used to directly send events to the
+// Amplitude HTTP API.
+type MetricsTracker struct {
+	client *http.Client
+	apiKey string
+	userID string
+	props  commonProps
+	start  time.Time
+}
+
+type event struct {
+	UserID    string      `json:"user_id"`
+	DeviceID  string      `json:"device_id"`
+	EventType string      `json:"event_type"`
+	Props     interface{} `json:"event_properties"`
```

Comment:
maybe as a separate PR once this is merged into master, can you see what additional properties (user properties and event properties) we can add on here to represent the official fields for things like OS, Country and other supported fields from Amplitude

Reply:
ref #516